docs/DeveloperGuide.md

```diff
@@ -382,6 +382,42 @@ Aspect: How to allow the `SortCommand` to sort using different possible orders w
     * Pros: Will avoid the need for creating multiple classes.
     * Cons: Seems redundant to use another switch statement for controlling the `SortCommand` behaviour after already using one in `SortCommandParser` for determining the order to use.
 
+
+### Remind Feature
+
+#### Implementation
+The remind feature allows the user to view a list of upcoming interviews within the next 1 week, sorted by interview date and time. <br>
+
```

Comment:
Just checking is this double line break intended?

Reply:
thanks for spotting this! I've removed it